src/main/java/graphql/execution/nextgen/FetchedValueAnalysis.java

```diff
@@ -190,6 +183,7 @@ public Builder executionStepInfo(ExecutionStepInfo executionInfo) {
 
         public Builder fetchedValue(FetchedValue fetchedValue) {
             this.fetchedValue = fetchedValue;
+            errors(fetchedValue.getErrors());
```

Comment:
this is not so great I think, because it kinda violates the expectation for a builder: if I set a new fetchValue it should be ok. 
the error should be add in the constructor or maybe in the `build` method imho

Reply:
fixed